app/controllers/api/v1/reports_controller.rb

```diff
@@ -0,0 +1,24 @@
+module Api
+  module V1
+    # reports endpoint.  No swagger docs since this is admin-user only
+    # This returns tsv files
+    class ReportsController < ApiBaseController
+      include ActionController::RequestForgeryProtection
+
+      before_action :authenticate_api_user!
+
+      def show
+        if !current_api_user.admin?
+          head 403 and return
```

Comment:
Technically, we don't validate that a user is an admin for the reports section.  We only validate that `current_user.acts_like_reporter?`, which returns `true` for admins or any other user who has `reporter` set to `true`.